test/runtests.jl

```diff
@@ -70,20 +84,16 @@ using Test
             f(a; kw=1) = a - kw
             a = 2.0
             kw = 1
-            call1 = @cached_call f(a; kw=kw)
-            @static if VERSION >= v"1.5"
-                call2 = @cached_call f(a; kw)
```

Comment:
this has been removed as it tests the same thing (`kw` syntactic sugar) as the above test